setup.py

```diff
@@ -1,44 +1,3 @@
-import os
-from setuptools import find_packages, setup
-from asgiref import __version__
+from setuptools import setup
 
-
-# We use the README as the long_description
-readme_path = os.path.join(os.path.dirname(__file__), "README.rst")
-
-
-setup(
-    name='asgiref',
-    version=__version__,
-    url='http://github.com/django/asgiref/',
-    author='Django Software Foundation',
-    author_email='foundation@djangoproject.com',
-    description='ASGI specs, helper code, and adapters',
-    long_description=open(readme_path).read(),
-    long_description_content_type='text/x-rst',
-    license='BSD',
-    zip_safe=False,
-    packages=find_packages(exclude=['tests']),
-    include_package_data=True,
-    python_requires=">=3.5",
-    extras_require={
-        "tests": [
-            "pytest~=4.3.0",
-            "pytest-asyncio~=0.10.0",
-        ],
-    },
-    classifiers=[
-        'Development Status :: 5 - Production/Stable',
-        'Environment :: Web Environment',
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: BSD License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
-        'Programming Language :: Python :: 3.8',
-        'Topic :: Internet :: WWW/HTTP',
-    ],
-)
+setup()
```

Comment:
If you want GitHub's dependency/security features to continue working for this project and projects that use it, you need to at least leave the `name` in `setup()`. (`install_requires` as well, but that's not applicable here.)

Reply:
All my projects use `setup.cfg` and get security alerts - not sure if they're passing on to others though.

Have you got a source for this?

Reply:
It's project that depend on this that wouldn't get alerts, because GitHub's dependency graph feature only understands `setup.py`, not `setup.cfg`. This has been my experience with all the Pallets projects.

Reply:
I had a look around and guessed it's parsed by their semantic, created this issue: https://github.com/github/semantic/issues/562

Reply:
OK, I'd like to preserve compatibility with that feature, so it's worth moving the `name` back in, I think.